src/js/thirdparty/ws.js

```diff
@@ -68,7 +68,7 @@ class BunWebSocket extends EventEmitter {
       } else if (event === "message") {
         this.#ws.addEventListener("message", ({ data }) => {
           const isBinary = typeof data !== "string";
-          if (isBinary) {
```

Comment:
This feels suspicious

Reply:
`ws.test.ts` posix SIGTERMs are from this